### PR TITLE
Bch master merge

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/app-data/abs/report-analyzer/npInterimNationalReport1.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿[
+<<<<<<< HEAD
```
Comment (randyhoulahan) [inline app/app-data/abs/report-analyzer/npInterimNationalReport1.json]: conflict committed

File: app/views/search/search-directive.js [hunk 2]
```diff
@@ -980,23 +980,29 @@ define(['app', 'text!views/search/search-directive.html','underscore', 'json!app
...
+                       
+                        
+                        _.each(realm.schemas, function(schema, key){
+                            if(!_.includes(['contact'], key)){
+                                addFilter(key,  {'sort': schema.sort,'type':schema.type,  'name':schema.title.en, 
+                                    'id':key, 'description':(schema.description||{}).en}); 
+                            }
+                        })
+                        // addFilter('focalPoint',  {'sort': 1,'type':'national',  'name':schemaNames.focalpoint, 'id':'focalPoint', 'description':'Institution designated to liaise with the Secretariat and make available information on procedures for accessing genetic resources and establishing mutually agreed terms, including information on competent national authorities, relevant indigenous and local communities and relevant stakeholders (Article 13.1).'});
```
Comment (randyhoulahan) [inline app/views/search/search-directive.js]: should just delete comments

File: app/views/directives/party-status.js [hunk 3]
```diff
@@ -2,52 +2,49 @@ define(['app', 'text!views/directives/party-status.html', 'js/common'], function
...
-					 function ($scope, $rootScope, $filter, $timeout, commonjs, $q) 
-					{
-					var iso_code;
+            controller: [ "$scope", '$rootScope', '$filter', '$timeout', 'commonjs', '$q', 'realm',
+            function ($scope, $rootScope, $filter, $timeout, commonjs, $q, realm) {
+                var iso_code;
+                
+                $scope.isAbs = realm.is('ABS');
+                $scope.isBch = realm.is('BCH');
+                // if($scope.code)
```
Comment (randyhoulahan) [inline app/views/directives/party-status.js]: delete

File: app/views/directives/party-status.js [hunk 4]
```diff
@@ -2,52 +2,49 @@ define(['app', 'text!views/directives/party-status.html', 'js/common'], function
...
+                    if(!code)return;
+                    code = code.toUpperCase();
+                    $q.when(commonjs.getCountry(code))
+                        .then(function(country){
+                            $scope.partyStatus = country;
+                        });
+                }
+                
+                $scope.$watch('code', function(newValue, oldValue){
+                    //if(newValue && newValue != oldValue){
```
Comment (randyhoulahan) [inline app/views/directives/party-status.js]: delete

File: app/views/countries/country-map.html [hunk 5]
```diff
@@ -10,33 +10,44 @@
...
+                z-index: 100;
+                max-width:200px;
+            }
         </style>
         
-        <style ng-if="options.mapType=='lmoDecisions'">
-
-            .amcharts-main-div, .amcharts-chart-div{
+        <style >
+            /* ng-if="options.mapType=='lmoDecisions'" */
```
Comment (randyhoulahan) [inline app/views/countries/country-map.html]: delete

File: app/views/forms/view/view-reference-records.directive.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
+﻿<div ng-if="referenceRecords" class="km-value">
+    <legennd>Reference records</legend>
```
Comment (randyhoulahan) [inline app/views/forms/view/view-reference-records.directive.html]: spelling

File: app/views/countries/country-map.html [hunk 7]
```diff
@@ -10,33 +10,44 @@
...
                 display: block;
-                float: right;
+                float: left;
                 bottom: 20px;
                 right: 5px;
                 cursor: pointer;
+                left:10%
             }
-            .amcharts-main-div, .amcharts-chart-div{
+            /* .amcharts-main-div, .amcharts-chart-div{
```
Comment (randyhoulahan) [inline app/views/countries/country-map.html]: delete

File: app/css/record.css [hunk 8]
```diff
@@ -125,9 +125,9 @@
         padding:0!important;
     }
 
-	#Record .record-body div {
+	/* #Record .record-body div {
```
Comment (randyhoulahan) [inline app/css/record.css]: delete

File: app/views/forms/view/view-reference-records.directive.html (hunk 6 above)
Comment (blaisefonseca) [reply app/views/forms/view/view-reference-records.directive.html]: fixed

**Pull request merged**